WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -264,7 +264,9 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
     
-    if (self.isScrollEnabled) {
+    if (self.isSidebarModeEnabled) {
+        _tableView = [[UITableView alloc] initWithFrame:CGRectZero style:UITableViewStyleGrouped];
```

Comment:
This will be the best approximation of the "sidebar" style that only exists natively in `UICollectionView` or `SwiftUI.List`. I don't think it will be feasible to replace `BlogDetailsViewController` in the scope of this project.